src/stores/risk.store.js

```diff
@@ -342,17 +346,378 @@ class RiskStore {
     return current_collateral_factor
   }
 
-  preformRecommendation = (recommendation) => {
-    // decrease ADA.e mint cap to 40
-    const [,asset, type, , , amount] = recommendation.split(' ')
-    for (let row of this.data) {
-      if(row.asset === asset){
-        row[`${type}_cap`] = amount
-        this.solve()
-        break;
-      }
+    preformRecommendation = (recommendation) => {
+        // decrease ADA.e mint cap to 40
+        const [, asset, type, , , amount] = recommendation.split(' ')
+        for (let row of this.data) {
+            if (row.asset === asset) {
+                row[`${type}_cap`] = amount
+                this.solve()
+                break
+            }
+        }
+    }
+
+    /**
+     * /////////// REVERSE SOLVER SANDBOX CODE \\\\\\\\\\\\\\\\\\\\
```

Comment:
only the lines below are new code (except in the init function). The change before are just prettier shenanigans